src/Microsoft.Health.Fhir.Shared.Api/Controllers/ConvertDataController.cs

```diff
@@ -29,6 +29,7 @@ namespace Microsoft.Health.Fhir.Api.Controllers
 {
     [ServiceFilter(typeof(AuditLoggingFilterAttribute))]
     [ServiceFilter(typeof(OperationOutcomeExceptionFilterAttribute))]
+    [Consumes("application/json")]
```

Comment:
Does this work with `application/fhir+json` too?

We have an attribute that goes through a lot of the checks with `[ServiceFilter(typeof(ValidateFormatParametersAttribute))]`

Reply:
> Does this work with `application/fhir+json` too?
> 
> We have an attribute that goes through a lot of the checks with `[ServiceFilter(typeof(ValidateFormatParametersAttribute))]`

Good point. It actually should accept both  "application/fhir+json" and "application/json" conforming this spec: https://hl7.org/fhir/parameters.html

It would be a good idea to add [ServiceFilter(typeof(ValidateFormatParametersAttribute))], but if only adding that would not mitigate this specific issue. The .net core parser gets hit first. Any idea?